index.js

```diff
@@ -62,7 +60,7 @@ async function main() {
     run = shouldContinue
   }
 
-  sdk.log(ux.colors.magenta(`\nSee you later! 👋`))
+  await ux.print(ux.colors.magenta(`\nSee you later! 👋`))
```

Comment:
How do you feel about  'See you next time! 👋' as the text, or maybe 'Thanks for using Incident.sh! 👋'